war/src/main/java/io/lumeer/engine/controller/SearchFacade.java

```diff
@@ -117,10 +117,10 @@
       } catch (CollectionNotFoundException e) {
          throw new InvalidQueryException("Search asks for collections that are not available: ", e);
       }
-
-      final DataDocument readRightsFilter = securityFacade.getReadRightsQueryFilter();
-      readRightsFilter.putAll(query.getFilters());
-      internalQuery.setFilters(readRightsFilter);
+      // TODO: What about access rights now? We don't use old SecurityFacade methods anymore
```

Comment:
You do not need to worry about access control in `SearchFacade`. I will refactor this class when I implement search in the front-end.